app/subscriber/src/features/home/constants/columns.tsx

```diff
@@ -1,27 +1,53 @@
-import { IContentModel, ITableHookColumn } from 'tno-core';
+import { ContentTypeName, IContentModel, ITableHookColumn } from 'tno-core';
 
 import { DetermineToneIcon } from '../utils';
 
-export const columns: ITableHookColumn<IContentModel>[] = [
-  {
-    name: 'tone',
-    label: 'TONE',
-    cell: (cell) => (
-      <DetermineToneIcon tone={cell.original.tonePools ? cell.original.tonePools[0]?.value : 0} />
-    ),
-  },
-  {
-    name: 'headline',
-    label: 'HEADLINE',
-    cell: (cell) => <div className="headline">{cell.original.headline}</div>,
-    width: 5,
-  },
-  {
-    name: 'sectionPage',
-    label: 'SECTION PAGE',
-    cell: (cell) => (
-      <div className="section">{`${cell.original.section}: ${cell?.original.page}`}</div>
-    ),
-    width: 1,
-  },
-];
+export const determinecolumns = (contentType: ContentTypeName) => {
+  let cols: ITableHookColumn<IContentModel>[] = [];
+  if (contentType === ContentTypeName.PrintContent) {
```

Comment:
Instead of duplicating the shared columns, just add the additional column when required.  It's easier to maintain.

Reply:
Updated!